configure.ac

```diff
@@ -1254,40 +1254,12 @@ dnl ncurses takes precedence.
 dnl
 AC_CHECK_HEADERS([curses.h curses/curses.h ncurses.h ncurses/ncurses.h])
 
-dnl Although n-library is preferred, only look for it if the n-header was found.
-CURSESLIBS=''
```

Comment:
I believe we still need to initialize the variable (or set it empty in the not-found case)

Reply:
We should initialize it before hand anyway.  It's possible in theory for it to not be set even in the found case if no library is required for curses support (like, it's a part of libc).  In practice that's never going to happen, but still.